### ISSUE TITLE
Add theme for overlay of dialog

### ISSUE THREAD

**Issue by vikshv** [react-toolbox/react-toolbox#1055]
Please add theme prop for overlay component in dialog component

**Comment by frolant** [comment]
Yes. It is very important. This theme is very much needed. Often there is a problem with settings for overlay in dialog component. Please add it..

**Comment by gsawasthi25** [comment]
Yes this is must have thing.

In the meanwhile has anyone found any workaround for the same ??

**Comment by javivelasco** [comment]
Added under the namespace `overlay`. If you want to style it just pass the properties of the theme namespaced to the `Dialog`. For example:

```css
.overlay {
  &.overlayOverlay {
    background-color: red;
  }
}
```

**Comment by grrseguin** [comment]
Hello,

I use the 1.2.3 version of `react-toolbox` and i can not custom the theme `prop` for `overlay` component in `Dialog` component.

The `overlay` theme is only available in the [dev branch](https://github.com/react-toolbox/react-toolbox/blob/13688edd9b28788c64932bbc5e2bec50b0f4ce55/components/dialog/Dialog.js#L66) isn't it? The [master branch](https://github.com/react-toolbox/react-toolbox/blob/master/components/dialog/Dialog.js#L55) don't have it, so my version too?

**Comment by javivelasco** [comment]
Sorry @grrseguin, we are in the process of deprecating 1.x so no further development will be made apart from accepting prs. You can safely use the dev branch which is released under 2.0-beta.x. It's released as a beta because there is a lot to do before officially releasing an stable but it's perfectly usable as stated in the ROADMAP
